### ISSUE TITLE
Accessibility issues with handbook layout

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-developer#368]
From [Alex Stine on Slack](https://wordpress.slack.com/archives/C04U953K77A/p1699932257988309?thread_ts=1699558771.994199&cid=C04U953K77A):

Here is a quick list:

- [x] The Improve it on GitHub link is not descriptive enough.
- [x] Same for See list of changes link.
- [x] The link NextGetting Started has no spacing and is announced as 2 words instead of 3.
- [x] The handbook chapters heading 2 is the first heading on the page. This design needs to be reworked, no questions asked.
- [x] The skip to main content link does not land you in the main content, it lands you in all the items before the main heading 1.
- [x] The main landmark should not wrap navigational elements such as the Chapters heading 2 or the search form above it.

**Comment by adamwoodnz** [comment]
cc @fcoveram @ndiego, not sure how hard this will be to fix yet

**Comment by ndiego** [comment]
> - The handbook chapters heading 2 is the first heading on the page. This design needs to be reworked, no questions asked.
> - The skip to main content link does not land you in the main content, it lands you in all the items before the main heading 1.
> - The main landmark should not wrap navigational elements such as the Chapters heading 2 or the search form above it.

Are these three items due to the fact that the Chapters list is inside `<main>` and not wrapped in an `<aside>` tag? I was comparing the current layout with other sites like [Mozilla](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Attribute_selectors) and the [React docs](https://react.dev/learn/reacting-to-input-with-state). In both cases, they have a heading that is not an H1 that is the first on the page, but, it's not in `<main>` and is wrapped in an `<aside>`.

cc @alexstine

**Comment by alexstine** [comment]
@ndiego It would be more valid if it were wrapped in an `<aside>` tag but it would be even better if the heading 1 came first. Headings should represent importance on a page so I always have argued that the `<aside>` should appear directly after `<main>` in the DOM.

**Comment by adamwoodnz** [comment]
Pretty sure I'll be able to reorder the content and keep the visual layout. It's next on my list after the ToC styling.

**Comment by adamwoodnz** [comment]
Should the Table of Contents also be an aside?

Something like

```html
<header { including breadcrumbs and search } >
<main>
<aside class="chapter-list">
<aside class="table-of-contents">
<footer>
```

The visual layout will be tough to achieve if so. It would help greatly if it was immediately after the h1 within main.

**Comment by alexstine** [comment]
@adamwoodnz I think they are both wrapped in `<nav>` by default? If this is the case, they don't need their own `<aside>`. There should really only ever be one `<aside>` on a page if possible.

**Comment by adamwoodnz** [comment]
> * The link NextGetting Started has no spacing and is announced as 2 words instead of 3.

Are the 'Previous' and 'Next' bits actually useful with a screen reader? I notice there is a `rel="next"` attribute on the anchor too.

Trying to decide whether to add punctuation (the design has a middot, but it's not implemented as text), or make the link content only the page title for screen readers.

**Comment by alexstine** [comment]
Next and previous are useful labels if it conveys specific actions. For example, Next section: section name or Next chapter: chapter name.

**Comment by adamwoodnz** [comment]
> * The Improve it on GitHub link is not descriptive enough.
> * Same for See list of changes link.

What would you suggest for these?

Help improve this document
See the history of this document

Visual for designers:

![Screenshot 2023-11-16 at 3 26 27 PM](https://github.com/WordPress/wporg-developer/assets/1017872/495de6cd-3856-436b-9186-8001ff2e1343)

**Comment by alexstine** [comment]
@adamwoodnz 

> The Improve it on GitHub link is not descriptive enough.

Improve on GitHub: doc title here

> Same for See list of changes link.

Document revisions: doc title here

It needs to convey current context.

**Comment by adamwoodnz** [comment]
The layout updates have just been deployed, GitHub links and pagination to follow.

**Comment by alexstine** [comment]
@adamwoodnz Would it be possible to also move the meta information inside the complementary landmark? It would likely be more discoverable just above the Chapters heading. Other than that, really much improved. Thanks.

**Comment by adamwoodnz** [comment]
> Would it be possible to also move the meta information inside the complementary landmark?

This will be difficult to achieve as visually the complementary landmark sits to the left of main in a sidebar. It will take quite a refactor, but could be possible.

[Issue](https://github.com/WordPress/wporg-developer/issues/400) created.
